docs/deploy.rst

```diff
@@ -208,18 +208,23 @@ openssl will then ask you a series of questions. You can enter whatever values a
 
 3. Decide whether you want to use python's built-in SSL library, or the pyOpenSSL library. CherryPy supports either.
 
-    a) *Built-in.* To use python's built-in SSL, add the following line to your CherryPy config:
+    a) *Built-in.* CherryPy uses Python's built-in SSL by default. To set it explicitly, set this line in your CherryPy config:
```

Comment:
It seems like there's at least one place where `builtin` is not the default:
* https://github.com/cherrypy/cherrypy/blob/8db3d07/cherrypy/_cpnative_server.py#L153C56-L153C67

Plus, there's another place that would need to be cleaned up for this docs change to make sense:
* https://github.com/cherrypy/cherrypy/blob/8db3d07/cherrypy/_cpwsgi_server.py#L86-L89